From: Daniel Mack <daniel@zonque.org>
Date: Tue, 24 Mar 2015 19:51:55 +0100
Subject: [PATCH] kdbus: Fix CONFIG_KDBUS help text

Drop a left-over from the times when documentation lived in a
simple text file, which is no longer the case. Mention the
auto-generated man-pages and HTML files instead.

Reported-by: Jiri Slaby <jslaby@suse.cz>
Signed-off-by: Daniel Mack <daniel@zonque.org>
Reviewed-by: David Herrmann <dh.herrmann@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 init/Kconfig | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/init/Kconfig b/init/Kconfig
index 02735f91836e..c4075075295d 100644
--- a/init/Kconfig
+++ b/init/Kconfig
@@ -268,10 +268,11 @@ config KDBUS
 	  D-Bus is a system for low-latency, low-overhead, easy to use
 	  interprocess communication (IPC).
 
-	  See Documentation/kdbus.txt
+	  See the man-pages and HTML files in Documentation/kdbus/
+	  that are generated by 'make mandocs' and 'make htmldocs'.
 
-	  To compile this driver as a module, choose M here: the
-	  module will be called kdbus.
+	  If you have an ordinary machine, select M here. The module
+	  will be called kdbus.
 
 config CROSS_MEMORY_ATTACH
 	bool "Enable process_vm_readv/writev syscalls"
